models/users.model.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection")
+
+exports.selectUsers = () => {
+    return db.query(
+        `SELECT * FROM USERS`)
+        .then((result) => {
+            return result.rows
+        })
+        .catch((err)=> {
+            console.error(err)
+            throw err
```

Comment:
We don't need to throw an error here. If we reach the catch block then our promise has already been rejected. Instead here we should invoke next with our error to pipe it into our error handling middleware